### PR TITLE
Tests are written for better coverage (62.8 vs 81.0 %) of the revoke pac...

### PR DESCRIPTION
**Pull request by ZenGround0**
...kage.  Additionally a few clarifying comments are included in the revoke file.

### CODE REVIEW

File: revoke/revoke.go [hunk 1]
```diff
@@ -58,10 +58,13 @@ func ldapURL(url string) bool {
 //	false, false: an error was encountered while checking revocations.
 //
 //	false, true:  the certificate was checked successfully and
-//		      it is not revoked.
+//		            it is not revoked.
```
Comment (lziest) [inline revoke/revoke.go]: The spacing in the comment need some consistency.

Comment (lziest) [inline revoke/revoke.go]: Otherwise looks good to me.

Comment (grittygrease) [inline revoke/revoke.go]: Wyatt, can you change this one line and squash it into your commit? Once that is done, we will merge.

**Comment by grittygrease** [comment]
LGTM

**Pull request merged**